recipes/crowcpp-crow/all/patches/001-disable-examples.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index f00e9a9..79f432d 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -29,7 +29,7 @@ if (MSVC)
+ 	set(Boost_USE_STATIC_LIBS "On")
+ 	find_package( Boost 1.52 COMPONENTS system thread regex REQUIRED )
+ else()
+-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -std=c++14 -pedantic -Wextra")
++	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -std=c++11 -pedantic -Wextra")
+ 	find_package( Boost 1.52 COMPONENTS system thread REQUIRED )
+ endif()
+ 
+@@ -42,7 +42,7 @@ set(PROJECT_INCLUDE_DIR
+ include_directories("${PROJECT_INCLUDE_DIR}")
+ include_directories("${PROJECT_SOURCE_DIR}")
+ 
+-add_subdirectory(examples)
++#add_subdirectory(examples)
```

Comment:
If I read it correctly, then you are part of this project right?

Could you please consider adding a CMake option like `BUILD_EXAMPLES` so that we don't need to patch it?

Reply:
I'm not part of the project. The patch addresses an issue in the library that has been solved after the release tag v0.2. Until there's no new release of the library, I'm skipping the build of the examples, leaving only the tests active.

Reply:
The fix you're talking about is to be able to build examples in all environments.
But conan cci recipes prefer not to build examples at all.

To make it easy for the recipe, it would be great having this `add_subdirectory(examples)` under `if (BUILD_EXAMPLES)` and we would just `BUILD_EXAMPLES` to False, instead of patching.

What I suggest:
- to make a PR in the crow project to add this variable (it may be ON by default, to not change the current behaviour)
- then ask the developer to release 0.2.1.

After that patch for examples will not be required at all.

It might be an overkill and/or too difficult, but it would make maintenance easier for the future versions.

Reply:
Yeah I understand your point, which I totally share. I will try to do my best

Reply:
I've submitted the PR in the repository, the project maintainers plan to do a release soon. I will update the recipe once they'll release a new version!